src/test/java/step1/OperatorTest.java

```diff
@@ -0,0 +1,46 @@
+package step1;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import step1.controller.StringCalculator;
+import step1.domain.Numbers;
+import step1.domain.Operator;
+import step1.domain.Operators;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class OperatorTest {
+
+
+    @ParameterizedTest
+    @ValueSource(strings = {"4 + 2"})
+    public void 덧셈_결과_검증(String input) {
+        assertThat(StringCalculator.calculate(input)).isEqualTo(getResult(input));
+    }
```

Comment:
```suggestion
public class OperatorTest {


    @ParameterizedTest
    @ValueSource(strings = {"4 + 2"})
    public void 덧셈_결과_검증(String input) {
        assertThat(Operator.Add.calculate(input)).isEqualTo(6);
    }
```
OperatorTest 테스트이니 Operator에게 맡기는 게 맞을 것 같아요
StringCalculator 테스트는 StringCalculatorTest에 위치하면 되겠네요!

Reply:
getResult() 메서드는 굳이 필요없는 메서드 일까요?

Reply:
넵 Operator의 기능을 검증하는 것이 목적이므로 
결과를 만드는 메소드는 딱히 필요없을 것 같아요